shell/platform/android/io/flutter/embedding/engine/FlutterEnginePluginRegistry.java

```diff
@@ -252,7 +253,8 @@ private boolean isAttachedToActivity() {
 
   @Override
   public void attachToActivity(@NonNull Activity activity, @NonNull Lifecycle lifecycle) {
-    Log.d(TAG, "Attaching to an Activity.");
+    Log.d(TAG, "Attaching to an Activity."
```

Comment:
nit: We want to guard these. I think arguably this should be verbose instead of debug too.

```
../../../flutter/shell/platform/android/io/flutter/embedding/engine/FlutterEnginePluginRegistry.java:256: Error: The log call Log.d(...) should be conditional: surround with if (Log.isLoggable(...)) or if (BuildConfig.DEBUG) { ... } [LogConditional]
    Log.d(TAG, "Attaching to an Activity."
```

Reply:
Ok, changed the Log.d's to Log.v's. Do we have a style doc that says when to use one logging level vs another?